packages/nuxt/src/pages/runtime/page.ts

```diff
@@ -8,6 +8,7 @@ import { toArray } from './utils'
 import type { RouterViewSlotProps } from './utils'
 import { generateRouteKey, wrapInKeepAlive } from './utils'
 import { RouteProvider } from '#app/components/route-provider'
+import ClientOnly from '#app/components/client-only'
```

Comment:
I wonder if we have any negative impact by now always importing `ClientOnly` 🤔

Reply:
I also had doubts about the size of the bundle, but the minimal size fixture seems to pass. Maybe we can have a build flag, that will enable/disable this feature, but then it won't be as seamless as is

Reply:
The minimal fixture doesn't use `<NuxtPage>` so it wouldn't be affected.